src/sdrf_pipelines/converters/openms/constants.py

```
@@ -1,97 +1,81 @@
+        expected_ids = list(range(1, len(channels) + 1))
+        if sorted(channels.values()) != expected_ids:
+            raise ValueError(f"Channel IDs for {plex!r} must be unique and contiguous from 1")
+        channel_map[plex] = channels
+
+    return channel_map
+
+
+def _load_channel_map() -> dict[str, dict[str, int]]:
+    """Load the canonical label-to-channel-ID mapping from the package data."""
+    resource = files("sdrf_pipelines.converters.openms").joinpath(_CHANNEL_MAP_RESOURCE)
+    document: object = yaml.load(resource.read_text(encoding="utf-8"), Loader=_UniqueKeyLoader)
```

Ruff (0.16.0)

[error] Probable use of unsafe loader `_UniqueKeyLoader` with `yaml.load`. Allows instantiation of arbitrary objects. Consider `yaml.safe_load`.

(S506)